packages/web-components/src/assets/scss/typography.scss

```diff
@@ -0,0 +1,218 @@
+:root {
+  --font-family: "PTRootUIWebVF", ui-sans-serif, system-ui, sans-serif, "Apple Color Emoji", "Segoe UI Emoji",
```

Comment:
Add docs to the readme, which fonts we have to install to reach the proper view of the components. And add notes, that we distribute the fonts in the package